data/menus.yaml

```diff
@@ -1,15 +1,24 @@
+- title: Community
```

Comment:
I think the Zulip and GitHub links need to be at the top. The others are much less important but they make sense in this section.